modules/channel-twilio/src/renderers/carousel.ts

```diff
@@ -0,0 +1,53 @@
+import * as sdk from 'botpress/sdk'
+import { ChannelRenderer } from 'common/channel'
+import { formatUrl } from 'common/url'
+import { TwilioContext } from '../backend/typings'
+
+export class TwilioCarouselRenderer implements ChannelRenderer<TwilioContext> {
+  get channel(): string {
+    return 'twilio'
+  }
+
+  get priority(): number {
+    return 0
+  }
+
+  get id(): string {
+    return TwilioCarouselRenderer.name
+  }
+
+  handles(context: TwilioContext): boolean {
+    return context.payload.items?.length
+  }
+
+  render(context: TwilioContext) {
```

Comment:
Can you add a comment explaining that we down-render carousels and why!

Reply:
+1